### ISSUE TITLE
could this module be used outside of Autoware?

### ISSUE THREAD

**Issue by sume-cn** [einsnull/ndt_gpu#1]
like NDT_OMP https://github.com/koide3/ndt_omp

**Comment by einsnull** [comment]
Yes, but I found that this model(ndt_gpu) is not as accurate as the ndt algorithm implemented in the PCL.

**Comment by AndrewBoWen666** [comment]
Hi, I just tried your NDT_GPU project, which was great. However, it showed great mismatch between input cloud and target cloud. Is this normal?
